script.js

```diff
@@ -48,3 +65,74 @@ function onSubmit() {
   const converted = amountIn * rate;
   amountOut.value = converted;
 }
+
+
+function searchFunction() {
```

Comment:
I like the behavior! but the search form should serve to filter currency based on [rates](https://github.com/HackYourFuture-CPH/js-projects/blob/main/projects/currency-converter/currency-converter.md#js2-week2---array-functions-arrow-function). eg: you have a table
-------------------
CZK|USD|0.2
DKK|CZK|0.5
USD|EUR|1.0
-------------------
And you want to print out results with rates between　(from)0.4 and (to)2.0 so the result would be
-------------------
DKK|CZK|0.5
USD|EUR|1.0
-------------------